rpc/src/tests/module/pool.rs

```diff
@@ -184,7 +184,8 @@ fn test_send_transaction_exceeded_maximum_ancestors_count() {
         });
     }
 
-    // the default value of pool config `max_ancestors_count` is 125, only 125 txs will be added to committed list of the block template
+    // the default value of pool config `max_ancestors_count` is 125,
```

Comment:
But I found:
https://github.com/nervosnetwork/ckb/blob/4851b307b018b6bc86d2f79632223c97364f53c6/resource/ckb.toml#L140-L139

the `max_ancestors_count` is 25.

Reply:
I think we can change the parameter in `ckb.toml` to 125 later.

Reply:
I think it's a comment error, we should change the default const value to 25 also.

Reply:
Should we change `const DEFAULT_MAX_ANCESTORS_COUNT: usize` to 25 too?

Reply:
it should be 125?
https://github.com/nervosnetwork/ckb/blob/b8ccf92e5c95310f6ea82a3c4575de7b05c03904/util/app-config/src/legacy/tx_pool.rs#L123

25 is originally referred from bitcoin, we use 125.

Reply:
I think it should be `min` instead of `max`?
```rust
 max_ancestors_count: cmp::min(DEFAULT_MAX_ANCESTORS_COUNT, max_ancestors_count), 
```

Reply:
no, it's `max`.
I think this commit is not complete, since we change from 25 to 125, we'd better also change the value in `ckb.toml`? 
https://github.com/nervosnetwork/ckb/commit/07d4db9e5ad4a9f0df079bf26e8aebd1c8e9b478

@zhangsoledad